packages/xo-server/src/xo-mixins/proxies.js

```diff
@@ -222,6 +222,7 @@ export default class Proxy {
       name: this._generateDefaultProxyName(date),
       vmUuid: vm.uuid,
     })
+    $defer.onFailure.call(this, 'unregisterProxy', id)
```

Comment:
Why not a simple `try`/`catch` on the instruction below?

Reply:
I choose to use defer for simplification. But i can also use `try/catch`

Reply:
You can leave it like that for this function :-)